Forta-Bots-Challenge-1/src/agent.spec.ts

```diff
@@ -0,0 +1,282 @@
+import { FindingType, FindingSeverity, HandleTransaction } from "forta-agent";
+import { Interface } from "ethers/lib/utils";
+import { createAddress } from "forta-agent-tools";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { provideHandleTransaction } from "./agent";
+import { BigNumber } from "ethers";
+import {
+  CREATE_AGENT_SIGNATURE,
+  DESTROY_AGENT_SIGNATURE,
+  FORTA_REGISTRY_ADDRESS,
+  NETHERMIND_FORTA_DEPLOYER_ADDRESS,
+  UPDATE_AGENT_SIGNATURE,
+} from "./constants";
```

Comment:
In testing, we don't typically use the same constants as we do in production. We create mock constants that are used exclusively in testing.

Reply:
This mostly applies to addresses, not functions.